.github/workflows/refresh_db.yml

```diff
@@ -3,8 +3,8 @@ name: DB Refresh (Objects API)
 on:
   schedule:
 #    - cron: '*/20 * * * *'  # every 20 minutes
-    - cron: '0 5 * * *'  # every day 5am GMT (12-1am EST/EDT)
-#    - cron: '0 * * * *'  # every hour on the hour
+#    - cron: '0 5 * * *'  # every day 5am GMT (12-1am EST/EDT)
+    - cron: '0 * * * *'  # every hour on the hour
```

Comment:
@Sigfried The last refresh took 40+ minutes. This is largely because of 3 new Hope test sets where it is actually able to fetch over 50k concepts (I have observed), but is still limited to 10k expressions (or maybe it's the reverse). But perhaps future runs could also take a long time, so I'm setting the refresh rate to 1x/hour instead of 1x/20min until we do some optimizations soon.